src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -677,6 +688,29 @@ private void processUnprovidedTypes(Set<String> provides, Set<String> transitive
     }
   }
 
+  /**
+   * If any inputs declare a legacy namespace, emit aliases for their exports in goog.module style.
```

Comment:
OOC, why it is better to emit both goog.provide and goog.module (through alias), vs just emit solely in goog.module style.

I have a hunch that codewise what you currently have is easier, but want to hear your opinion here.

Reply:
We have to emit both styles, since declareLegacyNamespace() allows you to import in a goog.provide() file without the goog.scope() and goog.module.get() workaround.  There's a more complicated thing we could do, which is replace the module$contents$foo initial definitions of the types with module$exports$foo definitions, but that'd be much harder than this.